ops/docker/Dockerfile.packages

```diff
@@ -73,7 +73,7 @@ WORKDIR /opt/optimism
 COPY --from=manifests /tmp/manifests  ./
 COPY *.json ./
 
-RUN pnpm install:ci
```

Comment:
I suspect we want the `--frozen-lockfile` option that `install:ci` had previously here to ensure it uses exactly the versions in the lock file rather than potentially upgrading some based on semver.

Reply:
Good catch. I updated with this. Attempting to build the image locally to make sure it builds properly, just taking awhile